lightsensor.lua

```diff
@@ -53,7 +53,7 @@ minetest.register_node("digilines:lightsensor", {
 	},
 	on_construct = function(pos)
 		local meta = minetest.get_meta(pos)
-		meta:set_string("formspec", "field[channel;Channel;${channel}]")
+		meta:set_string("formspec", "field[channel;"..S("Channel")..";${channel}]")
```

Comment:
Here and in `rtc.lua` too. Same situation.

Reply:
Fixed and fixed.